blog/02-15-2023-build-feature-rich-persistent-dapp-on-celo-using-wagmi/build-feature-rich-persistent-dapp-on-celo-using-wagmi.md

```diff
@@ -0,0 +1,747 @@
+---
+title: Build feature-rich, persistent dapp on celo using wagmi
+description: An alternative method of connecting to Celo networks
+authors:
+  - name: Isaac Jesse
+    title: Web3, Smart Contract Developer
+    url: https://github.com/bobeu
+    image_url: https://github.com/bobeu.png
+tags: [hardhat, celo, solidity, react, nextjs, materialui, typescript]
+hide_table_of_contents: true
+slug: /tutorials/build-feature-rich-persistent-dapp-on-celo-using-wagmi
+---
+
+![header](images/1.png)
+
+## Introduction
+
+Since Ethereum emerged, there has been a high demand for web3 developers. Unlike the way traditional applications are built, building for blockchain requires extra effort to give the users the experience they need to keep coming back. To build a good user interface, this article will introduce you to one of the many libraries that help you get started. 
+
+## Prerequisites​
+
+We will walk through one of the many ways of connecting and interacting with dapp on the Celo blockchain using wagmi. Celo blockchain is Ethereum compatible, and this indicates that we can use the same tools for building dapps on the Ethereum blockchain to build for Celo. This is an advanced tutorial, hence it is expected that you know the following:
```

Comment:
We will walk through one of the many ways of connecting and interacting with Dapp on the Celo blockchain using wagmi. Celo blockchain is Ethereum compatible, indicating that we can use the same tools for building Dapps on the Ethereum blockchain to build for Celo. This is an advanced tutorial.